util.js

```diff
@@ -240,11 +240,6 @@ exports.resolveFormatter = function(formatter) {
 		formatter =	CLIEngine.getFormatter(formatter) || formatter;
 	}
 
-	if (typeof formatter !== 'function') {
```

Comment:
Submitted in https://github.com/adametry/gulp-eslint/commit/fc9200e986faa2282145c8182c1aa142961bb7ec as your commit. Thanks.